src/main/python/rerank/scripts/interpolate.py

```diff
@@ -0,0 +1,123 @@
+'''scripts to aggregate the retrieval scores and the reranking scores by linear interpolation
+
```

Comment:
Please add a note here to say the input format... Is this standard TREC output format?
What's the output of this script? A ranking in the same format?